Source/WebCore/platform/graphics/cocoa/CMUtilities.mm

```diff
@@ -42,9 +45,12 @@
 #import "CoreVideoSoftLink.h"
 #import "VideoToolboxSoftLink.h"
 #import <pal/cocoa/AVFoundationSoftLink.h>
-#import <pal/cf/CoreMediaSoftLink.h>
 #import <pal/cf/AudioToolboxSoftLink.h>
+#import <pal/cf/CoreMediaSoftLink.h>
 
+#if ENABLE(VORBIS)
```

Comment:
Vorbis is always enabled on all platforms where we compile this file, so why do we need it in the file at all?

Reply:
I followed the pattern elsewhere, everything is conditional. We could probably remove it all, but that would be out of scope for this change.